gen_elements_elementGen.go

```diff
@@ -0,0 +1,2427 @@
+// Copyright (c) 2018 Paul Jolly <paul@myitcv.org.uk>, all rights reserved.
+// Use of this document is governed by a license found in the LICENSE document.
+
+// Code generated by elementGen. DO NOT EDIT.
```

Comment:
Whilst you're correct this can appear anywhere in the file (per https://golang.org/s/generatedcode) I have a slight preference for seeing this as the first line of a file, as a separate, standalone comment, after any build constraints. e.g. https://github.com/myitcv/react/blob/f32f0c5d7468766a687ac82cef960f5f64b72270/gen_AProps_reactGen.go#L1